guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p><s>In case of</s> <u>Once</u> the machine learning model <s>once it</s> is initialized with the data and the scikit-learn model parameters mentioned in the previous step, the <code>model.fit()</code> method is used for fitting the machine learning model. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>